src/java/org/apache/nutch/hostdb/UpdateHostDbReducer.java

```diff
@@ -88,95 +97,128 @@ public void configure(JobConf job) {
     numericFields = job.getStrings(UpdateHostDb.HOSTDB_NUMERIC_FIELDS);
     stringFields = job.getStrings(UpdateHostDb.HOSTDB_STRING_FIELDS);
     percentiles = job.getInts(UpdateHostDb.HOSTDB_PERCENTILES);
-    
+
     // What fields do we need to collect metadata from
     if (numericFields != null) {
       numericFieldWritables = new Text[numericFields.length];
       for (int i = 0; i < numericFields.length; i++) {
         numericFieldWritables[i] = new Text(numericFields[i]);
       }
     }
-    
+
     if (stringFields != null) {
       stringFieldWritables = new Text[stringFields.length];
       for (int i = 0; i < stringFields.length; i++) {
         stringFieldWritables[i] = new Text(stringFields[i]);
       }
     }
 
+    stringDeltaExpression = job
+        .get(UpdateHostDb.HOSTDB_UPDATEDB_DELTA_EXPRESSION);
+    if (!org.apache.commons.lang3.StringUtils.isEmpty(stringDeltaExpression)) {
+      // Create or retrieve a JexlEngine
+      JexlEngine jexl = new JexlEngine();
+
+      // Dont't be silent and be strict
```

Comment:
`Dont't` must be a typo, but beyond that, `setSilent(true)` seems to contradict this comment.

Reply:
@YossiTamari 
The funny part it is a copy-paste from ReadHostDb line 83.
How do you propose to fix it?

Reply:
I'm not sure what the original intent was, but maybe we should replace this whole code with: 
`this.deltaExpression = org.apache.nutch.util.JexlUtil.parseExpression(stringDeltaExpression);`?